package.json

```diff
@@ -27,6 +27,7 @@
   },
   "dependencies": {
     "handlebars": "^3.0.0",
+    "handlebars-helper-sri": "0.0.0",
```

Comment:
I usually start at 0.0.0, and wait a week or so to make sure I hadn't overlooked anything :)